src/west/app/project.py

```diff
@@ -845,6 +855,77 @@ def handle(group_filter_item):
 
         self.fs = self.fetch_strategy()
 
+    @staticmethod
+    def _update_concurrent_thread(args):
+        project, self, logdir = args
+        if logdir:
+            stdout = os.dup(sys.stdout.fileno())
+            stderr = os.dup(sys.stderr.fileno())
+
+            logfile = open(os.path.join(logdir, project.name), "wb")
+            os.dup2(logfile.fileno(), sys.stdout.fileno())
+            os.dup2(logfile.fileno(), sys.stderr.fileno())
+
+        try:
+            try:
+                self.update(project)
+                return True
+            except subprocess.CalledProcessError:
+                return False
+        finally:
+            if logdir:
+                sys.stdout.flush()
+                sys.stderr.flush()
+
+                os.dup2(stdout, sys.stdout.fileno())
+                os.dup2(stderr, sys.stderr.fileno())
+
+    def update_project_list(self, projects):
+        failed = []
+        updated = []
+
+        if self.args.jobs > 1:
+            # XXX: this is not picklable
+            self.parser = None
+
+            if self.args.unbuffered:
+                logdir = None
+                logdir_name = None
+            else:
+                logdir = tempfile.TemporaryDirectory()
+                logdir_name = logdir.name
```

Comment:
will do, keep in mind that:
- the directory get deleted as soon as `logdir` goes out og scope
- the logs may not appear in the file depending on both python and kernel buffering.

Reply:
I'll add the isatty info to the help of `--jobs` because that seems more important to the user of that option than it is to the reader of the code.

Reply:
> the directory get deleted as soon as logdir goes out og scope
> the logs may not appear in the file depending on both python and kernel buffering.

Understood. I think the most common reason why someone will want this information is because the download silently hangs because firewall/proxy/etc.